engine/userguide/storagedriver/device-mapper-driver.md

```diff
@@ -623,6 +623,16 @@ block device and other parameters to suit your situation.
     <output truncated>
     ```
 
+### Active the `devicemapper` after reboot
+
+If you reboot the host and found that docker service failed to start,
+an error shows `Non existing device docker-thinpool`, you need to re-active
```

Comment:
I think you mean "re-activate". Otherwise, LGTM.

Reply:
Thank you for your suggesting, my poor English....